tests/sentry/plugins/interfaces/test_releasehook.py

```diff
@@ -118,3 +126,110 @@ def test_minimal(self):
         assert commit_list[1].message == 'bar'
         assert commit_list[1].author.name == 'Joe^^'
         assert commit_list[1].author.email == 'joe@localhost'
+
+
+class SetRefsTest(TestCase):
+    """
+    tests that when finish_release is called on a release hook,
+    we try to get the previous commits based on the version ref
+    and that we create `ReleaseHeadCommit`s for the version
+    """
+    @patch('sentry.tasks.commits.fetch_commits')
+    def test_minimal(self, mock_fetch_commits):
+        project = self.create_project()
+        version = 'bbee5b51f84611e4b14834363b8514c2'
+        data_list = [
+            {
+                'id': 'c7155651831549cf8a5e47889fce17eb',
+                'message': 'foo',
+                'author_email': 'jane@example.com',
+            },
+            {
+                'id': '62de626b7c7cfb8e77efb4273b1a3df4123e6216',
+                'message': 'hello',
+                'author_name': 'Jess',
+            },
+            {
+                'id': '58de626b7c7cfb8e77efb4273b1a3df4123e6345',
+                'message': 'bar',
+                'author_name': 'Joe^^',
+            },
+            {
+                'id': 'bbee5b51f84611e4b14834363b8514c2',
+                'message': 'blah',
+                'author_email': 'katie@example.com',
+            },
+        ]
+        user = User.objects.create(email='stebe@sentry.io')
+        repo = Repository.objects.create(
+            organization_id=project.organization_id,
+            name=project.name,
+            provider='dummy',
+        )
+        ProjectOption.objects.set_value(
+            key='heroku:repository',
```

Comment:
this test should also probably move to sentry-plugins unless it can be more generic